Omex.sln

```diff
@@ -276,6 +280,22 @@ Global
 		{2D51316B-9B0D-4B54-B5E5-FB0CEDD8E791}.Release|Any CPU.Build.0 = Release|Any CPU
 		{2D51316B-9B0D-4B54-B5E5-FB0CEDD8E791}.Release|x64.ActiveCfg = Release|Any CPU
 		{2D51316B-9B0D-4B54-B5E5-FB0CEDD8E791}.Release|x64.Build.0 = Release|Any CPU
+		{4F6590D4-925E-4865-A19E-6AC4FEDF2A9F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
I think we can safely remove all the "Any CPU" entries in the solution. The projects are x64 anyway.

Reply:
No only SF related build for `x64` everything else is `AnyCPU`